app/src/hooks/userVendors.ts

```diff
@@ -48,20 +48,19 @@ const useVendors = (): Hooks => {
           firstName: splitName?.firstName,
           lastName: splitName?.lastName,
           email,
-          mobile,
+          mobile: mobile?.replace(" ", ""),
```

Comment:
if mobile is not optional shouldn't the undefined check come before sending all this through?